packages/core/src/storage-manager.ts

```diff
@@ -0,0 +1,45 @@
+// import AsyncStorage from '@react-native-async-storage/async-storage'
```

Comment:
this shouldn't be in core...

Reply:
Agreed. The logic that is mobile specific should live in the mobile app.